sdks/wasm/bindings.cs

```diff
@@ -79,69 +104,68 @@ static int UnBindJSObject(int js_id)
 
         }
 
-        static int BindJSObject(JSObject obj)
-        {
-
-            int js_id = obj.JSHandle;
-            if (js_id <= 0)
-                throw new JSException($"Invalid JS Object Handle {js_id}");
-
-            if (bound_objects.ContainsKey(js_id))
-                obj = bound_objects[js_id];
-            else
-                bound_objects[js_id] = obj;
-
-            return (int)(IntPtr)obj.Handle;
-        }
-
-        static int UnBindJSObjectAndFree(int js_id)
+        static void UnBindJSObjectAndFree(int js_id)
         {
             if (bound_objects.ContainsKey(js_id))
```

Comment:
Use TryGetValue like
```suggestion
            if (bound_objects.TryGetValue (js_id, out var obj)
```
To avoid an extra lookup here.  Also what is the new `bound_objects[js_id].RawObject = null;` looks redundant?

Reply:
Yes you are correct with the TryGetValue.  Will make those changes.

If I do not null them out I get the following intermittently:

`Uncaught RuntimeError: memory access out of bounds`